reinforcement_learning/rlclientlib/live_model_impl.cc

```diff
@@ -122,10 +124,19 @@ namespace reinforcement_learning {
     }
   }
 
+  int live_model_impl::init_trace(api_status* status) {
+    const auto trace_impl = _configuration.get(name::TRACE_LOG_IMPLEMENTATION, value::NULL_TRACE_LOGGER);
+    i_trace* plogger;
+    RETURN_IF_FAIL(_trace_factory->create(&plogger, trace_impl,_configuration, nullptr, status));
+    _trace_logger.reset(plogger);
+    TRACE_LOG(_trace_logger, "API Tracing initialized");
```

Comment:
>TRACE_LOG [](start = 4, length = 9)

I feel as though distinguishing between [info, warning, error] would be beneficial for consuming the logs later. Timestamp would be useful aswell

Reply:
I was considering it.  But since you brought it up, I will add it.  A bit weary of introducing a complete logging framework for a limited use case so want to be careful here.